include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
This doesn't need to be in #ifdef anymore actually, right? I'm thinking what would be better, have an undefined compiler error or a run-time error? Although I guess there won't be a runtime error, but an assignment to struct error that would be confusing, right?

Should we have an #else which will give a better error message when compiling without long long support?